### DISCUSSION TITLE
登录不了建好的halo博客

### DISCUSSION THREAD

**Discussion by t31-cpu** [halo-dev/halo#7176] [Q&A]
建好的halo博客已经登陆过了，第二次就登录不进去了，端口服务器已经放行，容器也是运行的，并没有其他端口占用此端口

**Comment by JohnNiang** [comment]
建议提供更多的信息，当前信息不足以判断导致问题的原因。

**Comment by longtengkeji** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。

**Comment by XiaoWoUi** [comment]
> 建好的halo博客已经登陆过了，第二次就登录不进去了，端口服务器已经放行，容器也是运行的，并没有其他端口占用此端口

肯定还是配置问题造成的
